.github/workflows/update_google_font.yml

```diff
@@ -0,0 +1,58 @@
+name: 'Update Google Fonts'
+
+on:
+  schedule:
+    # This runs once a week: https://crontab.guru/#0_0_*_*_0
+    - cron: '0 0 * * 0'
+  workflow_dispatch:
+
+env:
+
+jobs:
+  update-fonts:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ env.NODE_LTS_VERSION }}
+          check-latest: true
+
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 25
```

Comment:
This could probably be lower like `2`

Reply:
Actually on second thought, we might need to use `0` to fetch all so we can commit and push